control/tests/freqresp_test.py

```diff
@@ -271,3 +272,68 @@ def test_options(editsdefaults):
     # Make sure we got the right number of points
     assert numpoints1 != numpoints3
     assert numpoints3 == 13
+
+@pytest.mark.parametrize(
+    "TF, initial_phase, default_phase, expected_phase",
+    [pytest.param(ctrl.tf([1], [1, 0]),
```

Comment:
This initializes the systems in the collection phase of pytest. During that phase, the filter for the matrix warnings with `use_numpy_matrix` is not active yet:
https://travis-ci.org/github/python-control/python-control/jobs/752332682#L2031

Instead of `TF` you can parametrize two parameters `num`, `den` (or just `den` in this case) and then call `ctrl.tf()` inside the test functuin

Reply:
Hmm, the warnings are already in master, and I can not reproduce them on my local machine with master or your branch. Maybe I am wrong here.

Reply:
Okay, it was a different recently merged PR. See #496. The warnings are about `StateSpace` matrices, not something from `tf()`